### ISSUE TITLE
XR support

### ISSUE THREAD

**Issue by Utopiah** [dsrw/enu#28]
Could be interesting to live code in VR or AR, cf https://docs.godotengine.org/en/stable/tutorials/xr/setting_up_xr.html

**Comment by dsrw** [comment]
Agreed. It's not clear to me how coding in AR/VR should work, but it's something I really want to explore. Hopefully before the end of the year.

**Comment by Utopiah** [comment]
Well I have my own environment to explore that based on JavaScript https://git.benetou.fr/utopiah/text-code-xr-engine where I mix all inputs (hand tracking, keyboard, controller) including examples with nodal programming but... I also do not know how it should work! It's definitely fascinating to explore. IMHO the intersection of direct manipulation, e.g hand tracking to move single 3D object, and indirect, e.g code to move multiple objects, to e.g move N objects according to a motion or microgesture precisely is what is interesting. It personally led me to explore back the "usual" questions e.g graph dependency of functions or what keyword to define in a language so again without being to provide a solution it definitely is an interesting quest.
